components/atoms/Input/Input.masks.jsx

```diff
@@ -186,6 +192,7 @@ export const maskEnum = {
   CurrencyDecimal: {mask: currencyDecimalMask},
   CurrencyAllowNegative: {mask: currencyMaskAllowNegative},
   Number: {mask: numberMask},
+  percentageWithoutDecimal: {mask: percentageWithoutDecimalMask},
```

Comment:
```suggestion
  PercentageWithoutDecimal: {mask: percentageWithoutDecimalMask},
```
Capitalize p for consistency